modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPasteTest.ts

```diff
@@ -193,7 +193,7 @@ describe('webdriver.tinymce.core.paste.CopyAndPasteTest', () => {
       );
       await pCopyAndPaste(
         editor,
-        { startPath: [ 0, 0, 0 ], soffset: 0, finishPath: [ 0, 0, 0 ], foffset: 3 },
+        { startPath: [ 0, 0, 0 ], soffset: 0, finishPath: [ 0, 0, 0 ], foffset: tagName === 'code' ? 4 : 3 },
```

Comment:
content of code is slightly changed due to nodeChange-event. But only on `code`-blocks.